app/Importers/DataImporter.php

```diff
@@ -90,6 +94,11 @@ protected function deleteMissingProviders(string $providerName, array $existingC
             ->whereNotIn("city_id", $existingCityProviders)
             ->whereNot("created_by", "admin")
             ->get();
+
+        foreach ($cityProvidersToDelete as $cityProvider) {
+            event(new ChangeInFavoriteCityEvent($cityProvider->city_id, $providerName, "removed from"));
```

Comment:
```suggestion
            event(new ChangeInFavoriteCityEvent($cityProvider->city_id, $providerName, Action::Removed));
```
maybe create `Enum` class for things like that